conf/helmfile.d/0300.frontend.yaml

```diff
@@ -85,7 +85,7 @@ releases:
         MODEL_PREFIX: "models/"
         AWS_REGION: '{{ env "AWS_REGION" | default "us-east-1" }}'
         UPLOAD_PREFIX: "uploads/"
-        JOB_TYPES: "mesmer,segmentation,caliban"
+        JOB_TYPES: "mesmer,segmentation,caliban,polaris"
```

Comment:
Just want to clarify that we'll leave polaris off the production branch but leave it on master since we're taking it down, otherwise LGTM!